wp-content/themes/workday/less/inc/footer.less

```diff
@@ -0,0 +1,53 @@
+.footer-bg {
+    background: @black;
+    #site-footer {
+        #boilerplate {
+            padding-top: 50px;
+            padding-bottom: 15px;
+            border-top: 2px solid @gray;
+            font-family: @font-body;
```

Comment:
should be `@font-headline` based on the mockup in https://github.com/INN/umbrella-workdayminn/issues/3, I think?

But honestly, `@font-body` looks better here and let's roll with it.